spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/route/Route.java

```diff
@@ -169,6 +169,8 @@ public String toString() {
 
 		protected Map<String, Object> metadata = new HashMap<>();
 
+		protected boolean enableDefaultFilters = false;
```

Comment:
Why have this? Routes defined in properties don't have a choice.

Reply:
Why have this? In previous version,routes defined bu fluent api not support default filters,so need a switch to compatible previous  version.In 3.x.x version i think can default support default filters for all routes no matter how they defined.

Reply:
I see. It shouldn't be a property per route. It should just be a general setting.